js/src/features/blocks/bubbles/textBubble/components/TextBubble.tsx

```diff
@@ -6,11 +6,15 @@ import { PlateBlock } from './plate/PlateBlock'
 import { computePlainText } from '../helpers/convertRichTextToPlainText'
 import { clsx } from 'clsx'
 import { isMobile } from '@/utils/isMobileSignal'
+import { applyFilterPlainText } from '../helpers/applyFilterPlainText'
```

Comment:
We can skip the `applyFilterPlainText`, the applyFilterRichText does all thats needed!